lib/utils/simple_bloc_delegate.dart

```diff
@@ -0,0 +1,28 @@
+import 'package:flutter/material.dart';
```

Comment:
@irfanSilkron explain usage file ni

Reply:
Untuk debug/monitor changes dalam bloc, contoh nak tengok event apa yang di-triggered & camna state tu transition from one state to another.